starter-code/pizza.js

```diff
@@ -1 +1,43 @@
 // Write your Pizza Builder JavaScript in this file.
+$(document).ready(function() {
+  $("#btn-pep").click(function() {
+    $(".pep").toggle("pep");
+  });
+  $("#btn-mush").click(function() {
+    $(".mushroom").toggle("mushroom");
+  });
+  $("#btn-gPep").click(function() {
+    $(".green-pepper").toggle("green-pepper");
+  });
+  // Default settings
+  $("#btn-wSauce").text("Regular sauce");
+
+  $("#btn-crust").text("Crust");
+  $(".crust").removeClass("crust-gluten-free");
+
+  $("#btn-wSauce").click(function() {
+    if (
+      $(this)
+        .text()
+        .includes("White")
```

Comment:
It's preferable to select elements using their classes or other attributes instead of looking at their content.